python/python/lance/dataset.py

```diff
@@ -4696,6 +4696,7 @@ def write_dataset(
     auto_cleanup_options: Optional[AutoCleanupConfig] = None,
     commit_message: Optional[str] = None,
     transaction_properties: Optional[Dict[str, str]] = None,
+    data_bucket_uris: Optional[List[str]] = None,
```

Comment:
I feel "bucket" is a term very specific to S3, can we use something more generic